src/plugins/embeddable/public/lib/embeddables/i_embeddable.ts

```diff
@@ -18,6 +18,8 @@ export type { EmbeddableInput };
 export interface EmbeddableOutput {
   // Whether the embeddable is actively loading.
   loading?: boolean;
```

Comment:
There is a conceptual mismatch between this "status", which actually describes a transition (or activity) and the "rendered" status. We could instead have the following status:

* `mounted`
* `loaded`
* `rendered`

If we decide on doing this, that's something for a follow-up PR though.